_test/test_mpi_wrappers/test_cluster_wrapper.m

```diff
@@ -396,22 +401,32 @@ function test_set_user_env(obj)
             [cluster,~,clOb] = obj.get_cluster_wrapper_tester(3);
             path = cluster.common_env_var('MATLABPATH');
             pathes = split(path,pathsep);
-            assertTrue(ismember(necessary_path,pathes));
-            assertFalse(ismember(test_dir,pathes));
+            assertTrue(obj.path_belongs(necessary_path,pathes));
+            assertFalse(obj.path_belongs(test_dir,pathes));
 
             addpath(test_dir);
 
             cluster = cluster.add_user_path(pc);
 
             path = cluster.common_env_var('MATLABPATH');
             pathes = split(path,pathsep);
-            assertTrue(ismember(test_dir,pathes));
-            assertTrue(ismember(necessary_path,pathes));
+
+            assertTrue(obj.path_belongs(test_dir,pathes));
+            assertTrue(obj.path_belongs(necessary_path,pathes));
+
             % remove path first to avoid warinings
             clear clPath
         end
     end
     methods(Access= protected)
+        function is = path_belongs(~,folder,path)
+            if ispc()
```

Comment:
I don't understand why there is a system specific difference here. Why does MATLAB `ismember` not work on Windows? This raises huge warning flags to me

Reply:
because path on Windows is case-independent, but different functions return drive letter in the path in different case. `Ismember` considers it different while in fact it is the same.